web/src/main/webapp/typings/index.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference path="modules/rx-dom/index.d.ts" />
```

Comment:
none of the files in typings should be in the repo; please delete this folder altogether.